src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/order/history.yaml

```
@@ -59,13 +59,136 @@ sylius_twig_hooks:
+                priority: 600
+            company:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/company.html.twig'
+                priority: 500
+            city:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/city.html.twig'
+                priority: 400
+            postcode:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/postcode.html.twig'
+                priority: 300
+            country_code:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/country_code.html.twig'
+                priority: 200
+            province_code:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/province_code.html.twig'
+                priority: 100
+            province_name:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/province_name.html.twig'
+                priority: 0
+
+        'sylius_admin.order.history.content.sections.addresses.shipping_address':
+            header:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/header.html.twig'
+                priority: 200
+            details:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details.html.twig'
+                priority: 100
+        
+        'sylius_admin.order.history.content.sections.addresses.shipping_address.header':
+            title:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/header/title.html.twig'
+                priority: 0
+        
+        'sylius_admin.order.history.content.sections.addresses.shipping_address.details':
+            modification_status:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/modification_status.html.twig'
+                priority: 100
+            data:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/data.html.twig'
+                priority: 0
+        
+        'sylius_admin.order.history.content.sections.addresses.shipping_address.details.modification_status':
+            label:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/label.html.twig'
+                priority: 100
+            logged:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/logged.html.twig'
+                priority: 0
+        
+        'sylius_admin.order.history.content.sections.addresses.shipping_address.details.data':
+            first_name:
+                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/first_name.html.twig'
```

```suggestion
                template: '@SyliusAdmin/order/history/common/sections/addresses/address/details/data/first_name.html.twig'
```
This comment applies to all hookables of this hook